src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/ToDownloadProcessor.scala

```diff
@@ -54,7 +54,6 @@ trait ToDownloadProcessor extends ScorexLogging {
           case Some(bestHeaderAtThisHeight) =>
             val toDownload = requiredModifiersForHeader(bestHeaderAtThisHeight)
               .filter(m => filter(m._2))
```

Comment:
it's hard to read and understand what does the line `filter(m._2)` does. Firstly,`filter` param has the same name as `filter` function and it is confusing. Secondly it's not obvious what does `requiredModifiersForHeader` return, but this part would be fixed soon .